.github/test-spec.yml

```diff
@@ -503,9 +503,46 @@
   - "modules/lib/hostap/**/*"
 
 "CI-cloud-test":
+  - "include/caf/**/*"
+  - "include/date_time.h"
+  - "include/dfu/**/*"
+  - "include/dk_buttons_and_leds.h"
+  - "include/drivers/flash/**/*"
+  - "include/flash_map_pm.h"
+  - "include/modem/**/*"
+  - "include/net/download_client.h"
+  - "include/net/fota_download.h"
   - "include/net/nrf_cloud*"
+  - "include/net/rest_client.h"
+  - "lib/at_host/**/*"
+  - "lib/at_monitor/**/*"
+  - "lib/at_parser/**/*"
+  - "lib/at_shell/**/*"
+  - "lib/date_time/**/*"
+  - "lib/dk_buttons_and_leds/**/*"
+  - "lib/location/**/*"
+  - "lib/lte_link_control/**/*"
+  - "lib/modem_antenna/**/*"
+  - "lib/modem_attest_token/**/*"
+  - "lib/modem_info/**/*/"
+  - "lib/modem_jwt/**/*"
+  - "lib/modem_key_mgmt/**/*"
+  - "lib/nrf_modem_lib/**/*"
   - "samples/cellular/nrf_cloud_*/**/*"
+  - "subsys/app_event_manager/**/*"
+  - "subsys/bootloader/**/*"
+  - "subsys/caf/**/*"
+  - "subsys/dfu/**/*"
+  - "subsys/net/lib/download_client/**/*"
+  - "subsys/net/lib/fota_download/**/*"
   - "subsys/net/lib/nrf_cloud/**/*"
+  - "subsys/net/lib/nrf_provisioning/**/*"
+  - "subsys/net/lib/rest_client/**/*"
+  - "subsys/nrf_security/**/*"
+  - "subsys/partition_manager/**/*"
+  - "subsys/trusted_storage/**/*"
+  - "sysbuild/**/*"
```

Comment:
remove sysbuild -- if that is changed in a way that breaks things, all kinds of other tests will fail outside of our responsibility